simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/AlternateAddressFragment.kt

```diff
@@ -61,15 +58,11 @@ class AlternateAddressFragment : Fragment(), AdapterView.OnItemSelectedListener
 
     navController = Navigation.findNavController(requireActivity(), R.id.card_creator_nav_host_fragment)
 
-    arguments?.let {
-      addressType = AlternateAddressFragmentArgs.fromBundle(it).addressType
-      if (addressType == AddressType.WORK) {
-        binding.headerTv.text = getString(R.string.work_address)
-      } else {
-        binding.headerTv.text = getString(R.string.school_address)
-      }
+    if (bundle.addressType == AddressType.WORK) {
+      binding.headerTv.text = getString(R.string.work_address)
+    } else {
+      binding.headerTv.text = getString(R.string.school_address)
```

Comment:
You can move the `binding.headerTv.text` outside the `if`  condition and then you'd just be specifying the string resource in there

Reply:
Good call, but I'd rather leave this as is, because it came from upstream, and we don't use the `cardcreator` module in Palace. If I change it, it will invite conflicts in future merges, without gaining much.